lib/registration.pm

```diff
@@ -120,11 +123,11 @@ sub fill_in_registration_data {
                 }
             }
             send_key 'alt-n';    # next
-            while (check_screen('import-untrusted-gpg-key', 20)) {
+            while (check_screen('import-untrusted-gpg-key')) {
                 record_soft_failure 'untrusted gpg key';
                 send_key 'alt-t', 2;
             }
-            sleep 20;            # scc registration need some time
+            sleep 30;            # scc registration need some time
```

Comment:
can you make this sleep adaptive? Just sleeping 30 seconds for the sake of it does not sound like a good idea.